# 2 Numbers

Solution of excercise 2

1. Remember the format() function we have used in the last video?? Now pass 145 and ‘o’ in the function and see what it returned. Try to find out which representation it is.


In [3]:
format(145, 'o')

'221'

2. There is a circular pond in a village. This pond has a radius of 84meter. Can you find the area of the pond? (Bonus: If there is exactly 1.4 liter of water in a square meter, what is the total amount of the water in the pond?)

    Hint:   
`Circle Area = πr2`

In [4]:
pond_radius = 84
area = 3.14 * pond_radius ** 2


In [5]:
area

22155.84

**BONUS**

Because there is exactly 1.4 liter of water in 1 square meter. And we know that the total area of pond is `22155.84` square meter.

We can get the total amount of water if we multiply the given amount wih given area. 

In [6]:
per_sqt_meter_water = 1.4
total_water_amount = area * per_sqt_meter_water
total_water_amount

31018.176

3. 
If you cross a 490-meter-long street in 7 minutes, then what is your speed in meter per seconds. Print your answer without any decimal point in it.
        Hint:
        Speed = Distance / Time


In [8]:
time = 7 * 60
distance = 490
speed = distance // time
speed

1